README.md

```diff
@@ -374,13 +374,14 @@ cacert_content: |
 ```
 ## Python compatibility
 This library is tested with the following versions of Python:
-* Python 2.7
+* Python 2.7 (last version 0.10.0)
 * Python 3.6
+* Python 3.7
```

Comment:
Also Python 3.8 I guess (and even Python 3.9 I hope)?

Reply:
Done